admin/api/api.go

```diff
@@ -100,6 +111,9 @@ func (api *API) Handler() http.Handler {
 
 	r.HandleFunc("/", api.Index).Methods("GET")
 
+	r.HandleFunc("/workspaces/{workspace}/sync", api.Sync).Methods("POST")
+	r.HandleFunc("/workspaces/{workspace}/dump", api.Dump).Methods("POST")
```

Comment:
Can we add a prefix to make these APIs more literal?  like 

`/workspaces/{workspace}/config(declarative)/*`